test/parzival/accounts_test.exs

```diff
@@ -38,7 +38,7 @@ defmodule Parzival.AccountsTest do
   describe "get_user!/1" do
     test "raises if id is invalid" do
       assert_raise Ecto.NoResultsError, fn ->
-        Accounts.get_user!(-1)
+        Accounts.get_user!("11111111-1111-1111-1111-111111111111")
```

Comment:
If you are trying to achieve an UUID, you can generate one with `Ecto.UUID.generate/0`